core/src/com/unciv/ui/screens/victoryscreen/ReplayMap.kt

```diff
@@ -47,6 +47,7 @@ abstract class IndependentMiniMap(
     protected open fun includeTileFilter(tile: Tile): Boolean = true
 
     private fun createReplayMap(tileSize: Float): List<MinimapTile> {
+        @Suppress("EmptyFunctionBlock")
         val doNothing = fun(){}
```

Comment:
same here. MinimapTile(tile, tileSize, doNothing) could just be instantiated without doNothing right?